src/runtimes/go/index.ts

```diff
@@ -138,7 +137,7 @@ const zipFunction: ZipFunction = async function ({
     config,
     displayName: config?.name,
     generator: config?.generator || getInternalValue(isInternal),
-    runtimeVersion: featureFlags.zisi_golang_use_al2 ? 'provided.al2' : undefined,
+    runtimeVersion: 'provided.al2',
```

Comment:
We'll be able to remove this as well, once `provided.al2` is the default in our upload code. That'll have to wait until November 1st, though: https://answers.netlify.com/t/under-the-hood-changes-to-go-function-uploads-please-update-your-netlify-cli-version/104726